content/general-docs/introduction.adoc

```diff
@@ -160,4 +160,9 @@ Notes, footnotes, and examples shall not contain requirements or any information
 
 The following deliverables are provided for {THIS_STANDARD}:
 
-> Define the deliverables of this project here
+* ASAM OpenMATERIAL 3D Specification
+* https://github.com/asam-ev/OpenMATERIAL/tree/main/schemas[JSON schema files] for all json formats
```

Comment:
Also for the other deliverables is common to write in parentheses where the files are contained)
Whether they are contained in the same site, or in the deliverables menu of the site,...) So, maybe in our case we can write something like: (contained in the open-source standard repository)

Reply:
I added it as suggested.